aws_auditing_list.py

```diff
@@ -51,385 +81,333 @@ def initializeResourceDict(self, regions):
-        "st1_cost" : st1_price,
-        "standard_cost" : standard_price,
-        "gp2_size": gp2_vol_size,
-        "io1_size": io1_vol_size,
-        "sc1_size": sc1_vol_size,
-        "st1_size": st1_vol_size,
-        "standard_size": standard_vol_size
-        }
-
-# #   Get all snapshots and assign them to their volume
-#     # print("Collecting EBS snapshots")
-    for snapshot in snapshots['Snapshots']:
-        snap = snapshot['VolumeId']
-        if snap in recource_dict[region_name]["EBS"]:
-            recource_dict[region_name]["EBS"][snap]["snapshots"].append(snapshot['SnapshotId'])
-            if not snap in snap_vol:
-                snap_vol.append(snap)
-                snapshot_count = snapshot_count + 1
-        else:
-            recource_dict[region_name]["EBS"]["orphaned_snapshots"].append(snapshot['SnapshotId'])
-            orphaned_snapshot_count = orphaned_snapshot_count + 1
-
-    snap_vol_region[region_name] = snap_vol
-    snapshot_count_region[region_name] = {
-        "sc": snapshot_count,
-        "osc": orphaned_snapshot_count
-    }
-
-    # Get all instances for region_name and store in dict
-#     print("Collecting EC2 recource")
-    instance_ids = []
-    instance_list = ec2.describe_instances()
-    for r in  instance_list['Reservations']:
-        for i in r["Instances"]:
-            #instance_information = d['Instances']
-            id = i['InstanceId']
-            instance_ids.append(i['InstanceId'])
-            if "KeyName" in i:
-                key_name = i["KeyName"]
-            else:
-                key_name = ""
-            recource_dict[region_name]["EC2"][id] = {
-                "type": i['InstanceType'],
-                "key_name": key_name,
-                "launch_time": i['LaunchTime'],
-                "state": i['State']['Name'],
-                "volumes": []
+        for region_name in regions:
+            snap_vol = []
+            snap_vol_region[region_name] = {}
+            vol_cost_region[region_name] = {}
+            snapshot_count_region[region_name] = {}
+            count_list = {}
+            count = 0
+            snap_vol = []
+            orphaned_snap_vol = []
+            tv = 0
+            tvc = 0
+            snapshot_count = 0
+            orphaned_snapshot_count = 0
+            snapshot_count_list = {}
+            total_volume_cost = 0
+            attached_instances = 0
+            unattached_instances = 0
+            total_instances = 0
+            total_elb_cost = 0
+            total_elbv2_cost = 0
+            standard_count = 0
+            gp2_count = 0
+            io1_count = 0
+            sc1_count = 0
+            st1_count = 0
+            gp2_price = 0
+            io1_price = 0
+            sc1_price = 0
+            st1_price = 0
+            standard_price = 0
+            gp2_vol_size = 0
+            io1_vol_size = 0
+            sc1_vol_size = 0
+            st1_vol_size = 0
+            standard_vol_size = 0
```

Comment:
All of these don't need to be initialized, we should be able to do it when we create them.